de/flying/geofence.md

```diff
@@ -41,5 +41,9 @@ In summary:
 1. Once finished, click on the **Upload** button (top right) to send the fence (along with rally points and mission) to the vehicle.
 1. Set the breach action in the [GeoFence Failsafe](../config/safety.md#geofence-failsafe).
 
+:::note Geofences that do not include the Home position will be rejected by the flight controller.
```

Comment:
This is a change to a german doc. Needs to be removed from here and moved into the /en tree.

Reply:
thanks, done!